.vscode/settings_recommended.json

```diff
@@ -0,0 +1,30 @@
+{
+
+    // ========== Editor ==========
+    "files.insertFinalNewline": true,
+    "files.trimFinalNewlines": true,
+    "files.trimTrailingWhitespace": true,
```

Comment:
Hi @XuehaoSun, if pre-commit already covers it, we can remove it.

Reply:
Pre-commit already handles this.

Reply:
Thanks, removed these settings.

Reply:
@XuehaoSun , the "*.md" was not included in the current configuration; suggest to add it a separate PR.
https://github.com/intel/neural-compressor/blob/7763ed08c07d9e048467429b1b727e254db26665/.pre-commit-config.yaml#L47-L48